javers-core/src/test/groovy/org/javers/core/diff/appenders/OptionalChangeAppenderTest.groovy

```diff
@@ -9,6 +9,8 @@ import spock.lang.Unroll
 import static java.util.Optional.*
```

Comment:
Javers can be run on JDK 7 and java.util.Optional was introduced in JDK 8. Please use our internal org.javers.common.collections.Optional instead